### PR TITLE
Assignment submission

### PR DESCRIPTION
**Pull request by rckeely2**
Apologies... I did it the same day and forgot to do the pull request...

### CODE REVIEW

File: exercises.ipynb [hunk 1]
```diff
@@ -172,22 +319,230 @@
...
+    "    rates_df = pd.DataFrame.from_dict(rates, orient=\"index\")\n",
+    "    assets_new_df = pd.merge(assets_df,\n",
+    "                         rates_df,\n",
+    "                         how=\"left\",\n",
+    "                         left_on=\"curr\",\n",
+    "                         right_index=True)\n",
+    "    assets_new_df.columns = [\"value\", \"curr\", \"exchange_rate\"]\n",
+    "    assets_new_df[base_curr] = assets_new_df[\"value\"] / assets_new_df[\"exchange_rate\"]\n",
+    "    assets_new_df.drop(columns=\"exchange_rate\", inplace=True)\n",
+    "    assets_new_df.to_csv(output_file, compression='gzip', index=False)\n"
```
Comment (nandanrao) [inline exercises.ipynb:364]: Not sure why you chose to store this in an intermediate file, if you're worried about the API call just cache it! Less state, less places for things to go wrong.

**Review by nandanrao — COMMENTED** [review]
Great!

**Comment by rckeely2** [comment]
Because I is old and we didn't have all those fancy things when I was
learning how to do things. I put all kinds of things in files... I shall
endeavour to learn all these fancy tricks...

**Comment by nandanrao** [comment]
lol. very good!

**Pull request closed without merging**